src/dev/flang/tools/fzjava/FZJavaOptions.java

```diff
@@ -78,6 +96,19 @@ public class FZJavaOptions extends ANY
    */
   boolean _overwrite = false;
 
+
+  /**
+   * Value of property with name FUZION_HOME_PROPERTY.  Used only to initialize
+   * _fuzionHome.
+   */
+  private String _fuzionHomeProperty = System.getProperty(FUZION_HOME_PROPERTY);
+
+
+  /**
+   * Home directory of the Fuzion installation.
+   */
+  Path _fuzionHome = _fuzionHomeProperty != null ? Path.of(_fuzionHomeProperty) : null;
+
```

Comment:
This code and the declaration of FUZION_HOME_PROPERTY is duplicated from `Fuzion.java`. Would be better to extract this into a new clazz `FuzionHome` and use that both here and in `Fuzion.java`.